fga-eps-mds/2020.1-stay-safe-front-end
# Improve neighborhood review flow

## Issue 
Resolves #75 


## Description

Refactored the Review screen using a card that can be expandable to show the neighborhood ratings

## Screenshots:


![Screenshot from 2020-11-13 18-44-26](https://user-images.githubusercontent.com/52207400/99124547-285d3b00-25e1-11eb-8bee-8c53f22197c9.png)

## Checklist:



## How to validate?

1. Filter a crime to show the HeatMap.
2. Select a city.
3. Click in "Visualizar Bairros" of that city.
4. Check if the cards works fine opening more than one at a time and if the layout is good.
